data/presets/man_made/crane/portal_crane.json

```
@@ -0,0 +1,19 @@
+{
+    "icon": "temaki-crane",
+    "geometry": [
+        "point",
+        "area"
```

Is there a particular reason why this crane shouldn't be allowed on way nodes?

```suggestion
        "point",
        "vertex",
        "area"
```